pramen-py/tests/resources/real_config.yaml

```diff
@@ -45,3 +47,7 @@ metastore_tables:
     start: 2017-01-29
   name: table_out1
   path: /tmp/dummy/table_out1
+  reader_options:
+    compression: snappy
+  writer_options:
+    compression: snappy
```

Comment:
One tiny suggestion. In order to catch a situation where reader and writer options are mixed up (e.g. writer options are used to read a table or the other way around), it would be better to have a different set of options for the reader and for the writer.